src/components/TransactionBlock/Icons/Icon.tsx

```diff
@@ -1,24 +1,26 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
```

Comment:
damn this quite affects the codebase, maybe you can switch your workspace to double quotes for imports? would be good to make these moves one by one lol